src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -85,12 +86,15 @@ public void afterPropertiesSet() throws Exception {
     execute();
   }
 
+
   void execute() {
     if (!isEnabled()) {
       logger.info("Mongock is disabled. Exiting.");
       return;
     }
 
+    Object o = getObject();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "o" which hides the field declared at line 117. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "o". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "o" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)